README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://travis-ci.org/dbtsai/python-mimeparse.svg?branch=master
+   :target: https://travis-ci.org/dbtsai/python-mimeparse
+
+This module provides basic functions for handling mime-types. It can
+handle matching mime-types against a list of media-ranges. See section
+5.3.2 of the HTTP 1.1 Semantics and Content specification [RFC 7231] for
+a complete explanation: https://tools.ietf.org/html/rfc7231#section-5.3.2
+
+Functions
+=========
+
+``parse_mime_type()``
+----------------------
+
+Parses a mime-type into its component parts.
```

Comment:
Aside from a description, I think it would be helpful to show an example input/output for each function, e.g.:

> Parses a mime-type into its component parts. For example:
> ```
> > mimeparse.parse_mime_type('application/xhtml;q=0.5')
> ('application', 'xhtml', {'q': '0.5'}) 
>```